tests/integration/components/ui-message-test.js

```diff
@@ -0,0 +1,317 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('ui-message', 'Integration | Component | ui-message', {
+  integration: true,
+});
+
+test('it renders properly', function(assert) {
+  assert.expect(2);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check wrapper <div>.
+  assert.strictEqual($component.hasClass('ui'), true, 'Component\'s wrapper has \'ui\' css-class');
+  assert.strictEqual($component.hasClass('message'), true, 'Component\'s wrapper has \' message\' css-class');
+});
+
+test('size renders properly', function(assert) {
+  assert.expect(8);
+
+  // Render component.
+  this.render(hbs`{{ui-message
+    class=class
+  }}`);
+
+  // Retrieve component.
+  let $component = this.$().children();
+
+  // Check component's additional CSS-classes.
+  let additioanlCssClasses = 'small large huge massive';
+  this.set('class', additioanlCssClasses);
```

Comment:
Test for 'size' must use conponent's 'size' property of component instead of common property 'class'.
Like that:
`{{ui-message size="small"}}` & assert($component.hasClass('small'), true, ...)